engines/bops_admin/app/controllers/bops_admin/informatives_controller.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module BopsAdmin
+  class InformativesController < ApplicationController
+    before_action :set_informatives, only: %i[index]
+    before_action :build_informative, only: %i[new create]
+    before_action :set_informative, only: %i[edit update destroy]
+
+    rescue_from Pagy::OverflowError do
+      redirect_to informatives_path
+    end
+
+    def index
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def new
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def edit
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    def create
+      respond_to do |format|
+        if @informative.save
+          format.html do
+            redirect_to informatives_path, notice: t(".informative_successfully_created")
+          end
+        else
+          format.html { render :new }
+        end
+      end
+    end
+
+    def update
+      respond_to do |format|
+        if @informative.update(informative_params)
+          format.html do
+            redirect_to informatives_path, notice: t(".informative_successfully_updated")
+          end
+        else
+          format.html { render :edit }
+        end
+      end
+    end
+
+    def destroy
+      respond_to do |format|
+        if @informative.destroy
+          format.html do
+            redirect_to informatives_path, notice: t(".informative_successfully_destroyed")
+          end
+        else
+          format.html do
+            redirect_to informatives_path, alert: t(".informative_unsuccessfully_destroyed")
+          end
+        end
+      end
+    end
+
+    private
+
+    def set_informatives
+      @pagy, @informatives = pagy(current_local_authority.informatives.all_informatives(search_param), items: 10)
+    end
```

Comment:
If this is only being used in one action I wonder if it would be clearer just to include it in the `index` method directly, rather than as a hook? But not vital.